### DISCUSSION TITLE
Visualizing Network Nodes - Showcase

### DISCUSSION THREAD

**Discussion by avarga** [inet-framework/inet#611] [Showcases]
Discuss the "Visualizing Network Nodes" showcase here. Ideas and recommendations are welcome.

**Comment by johns-thomas** [comment]
I am trying simulate 3D scenario of communicating nodes. But  I don't know how to show ongoing transmissions, ongoing receptions, propagating radio signals (transmission of packets). Could you please help me?

**Reply by avarga** [reply]
AFAIK not many of the visualizers are available in 3D.

**Comment by guglielmo-morandin** [comment]
It would be very cool if packets could be shown on wired networks with OSG.
This https://youtu.be/BO0QhaxBRr0  was done manually, but I hope you can see the value of something like it. Basically a 3d version of what can be already done with omnet could be more valuable in a big network topology.
